### PR TITLE
RabbitMQ Management

### PR DESCRIPTION
**Pull request by bharel**

### CODE REVIEW

File: yellowbox/extras/rabbit_mq.py [hunk 1]
```diff
@@ -47,15 +47,32 @@ def stop(self):
         if self._auto_remove:
             self.container.remove()
 
+    def management_address(self):
+        try:
+            return f"http://localhost:{get_ports(self.container)[RABBIT_HTTP_API_PORT]}"
```
Comment (bentheiii) [inline yellowbox/extras/rabbit_mq.py:52]: rename to `management_url`?

Comment (bharel) [reply yellowbox/extras/rabbit_mq.py:52]: sure

Comment (bharel) [reply yellowbox/extras/rabbit_mq.py:52]: changed

**Review by bentheiii — APPROVED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -19,6 +19,7 @@ kafka-python = {version="*", optional=true}
 [tool.poetry.dev-dependencies]
 pytest = "^5.4.3"
 pytest-cov = "^2.10"
+requests = "^2"
```
[the remark this answers is not in the document]
Comment (bharel) [reply pyproject.toml:22]: 🤦 It's used in one line of testing of our testing framework.

**Pull request merged**